components/Information.tsx

```diff
@@ -94,10 +94,12 @@ export const Information: React.FC<Props> = ({ synth }) => {
         </Row>
         {synth.type === "emp" && (
           <>
-            <Row>
-              <div>Identifier Price:</div>
-              <div>{synth.identifierPrice}</div>
-            </Row>
+            {synth.identifierPrice && (
+              <Row>
+                <div>Identifier Price:</div>
+                <div>{ethers.utils.formatEther(synth.identifierPrice)}</div>
```

Comment:
I dont think it blocks this pr but we might want @Tommy1231232 and jesper to weigh in on max decimals showing

Reply:
Same -- I think we need a comprehensive solution because this is an issue in a bunch of places.